gix-date/src/lib.rs

```diff
@@ -29,6 +32,24 @@ pub struct Time {
     pub sign: time::Sign,
 }
 
+impl Sub for Time {
+    type Output = i64;
+
+    fn sub(self, rhs: Self) -> Self::Output {
+        self.total() - rhs.total()
+    }
+}
+
+impl Time {
+    fn total(&self) -> i64 {
+        if self.sign == Sign::Minus {
```

Comment:
The sign is just for serialization, what counts here is if `seconds` itself is negative.